src/Akka.Streams.Kafka.Tests/KafkaFixture.cs

```diff
@@ -0,0 +1,165 @@
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Akka.Util;
+using Docker.DotNet;
+using Docker.DotNet.Models;
+using Xunit;
+
+namespace Akka.Streams.Kafka.Tests
+{
+    [CollectionDefinition(Name)]
+    public sealed class KafkaSpecsFixture : ICollectionFixture<KafkaFixture>
+    {
+        public const string Name = "KafkaSpecs";
+    }
+    
+    public class KafkaFixture : IAsyncLifetime
+    {
+        private const string KafkaImageName = "confluentinc/cp-kafka";
+        private const string KafkaImageTag = "latest";
```

Comment:
@IgorFedchenko let's do this in a separate PR, but I would lock down the Docker image tag version to something specific - I'm looking at doing this with some of our other projects too. It's not an issue for most updates to the Docker image versions, but when there's a major release that includes some breaking changes that can cause CI to fail by accident. We should require future users to have to do that intentionally, usually at the same time as driver upgrades. Does that make sense?

Reply:
Sure, you are right. Actually, I was going to have it fixed for the same reason, but saw that "latest" tag is used for SQLServer fixture in reference project, so made it the same way.